server/hotrod/src/main/java/org/infinispan/server/hotrod/logging/Log.java

```diff
@@ -116,4 +117,8 @@ public interface Log extends BasicLogger {
    @LogMessage(level = WARN)
    @Message(value = "Client %s keeps providing outdated topology %s", id = 28029)
    void clientNotUpdatingTopology(SocketAddress socketAddress, int topologyId);
+
+   @LogMessage(level = TRACE)
```

Comment:
We don't need this for TRACE level, only INFO and higher are internationalized.

Reply:
Updated.